build/ci/phase-template.yml

```diff
@@ -36,5 +36,25 @@ phases:
         testRunTitle: Machinelearning_Tests_$(_phaseName)_$(_configuration)_$(Build.BuildNumber)
         configuration: $(_configuration)
         mergeTestResults: true
+    - task: CopyFiles@2
+      displayName: Stage build logs
+      inputs:
+        sourceFolder: $(Build.SourcesDirectory)
+        contents: 
+          ?(msbuild.*|binclash.log|init-tools.log)
+        targetFolder: $(Build.ArtifactStagingDirectory)
+    - task: CopyFiles@2
+      displayName: Stage more build logs
```

Comment:
Maybe stage test output?

Reply:
Sound good, I'll use that description.  

By the way, The only reason I have this second CopyFiles task is that I can't figure out the syntax to allow both of these being used in the same task.  I have a pending question on it here: https://stackoverflow.com/questions/53163581/how-to-copy-files-from-multiple-folders-on-azure-devops-ci

Do you know how to put both of these together?

Reply:
I just saw this reply. They only that I can think is the suggestion I just posted below: https://github.com/dotnet/machinelearning/pull/1527#discussion_r230968020

If that doesn't work, then we can leave it as is and wait to see if someone replies in the SO question or we could ask some Azure DevOps folks, or even open an issue in their tasks repo: https://github.com/Microsoft/azure-pipelines-tasks